mixer/adapter/stdio/zap.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stdio // import "istio.io/istio/mixer/adapter/stdio"
+
+import (
+	"time"
+
+	"github.com/natefinch/lumberjack"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"os"
```

Comment:
this should be a fmt / lint error. "os" should be in the standard block.

Reply:
I thought this was normally a lint error, but apparently not.

Fixed now anyway.